deeds-dapp-webapp/src/main/webapp/vue-app/dapp/farming/components/ComethLiquidityPool.vue

```diff
@@ -31,10 +31,10 @@
             class="mx-auto addLiquidityIcon"
             alt="Cometh Icon">
         </v-card-text>
-        <v-card-title class="justify-center pt-0 text-center text-break">
+        <v-card-title class="justify-center pt-0 text-center text-break headline font-weight-medium">
           {{ $t('rentLiquidityOnCometh') }}
         </v-card-title>
-        <v-card-subtitle class="justify-center pt-0 text-center text-break">
+        <v-card-subtitle class="justify-center pt-0 text-center text-break subtitle-1">
```

Comment:
It's may be better to define generic CSS for `.v-card-subtitle` to use a different font-size in all dapp.

Reply:
done!